shell/platform/fuchsia/flutter/accessibility_bridge_unittest.cc

```diff
@@ -294,7 +437,7 @@ TEST_F(AccessibilityBridgeTest, BatchesLargeMessages) {
   RunLoopUntilIdle();
 
   EXPECT_EQ(0, semantics_manager_.DeleteCount());
-  EXPECT_EQ(4, semantics_manager_.UpdateCount());
+  EXPECT_EQ(5, semantics_manager_.UpdateCount());
```

Comment:
I don't think these should be changing.

We should be resetting the state of the semantics_manager_ in SetUp for these tests.